src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  margin-bottom: 46.5px;
+  font-weight: $small-font-weight;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: $card-widht-size;
+
+  &__product-img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+    font-weight: $medium-font-weight;
+    font-size: $small-font-size;
+    line-height: $medium-line-height;
+    color: $color-light-black;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: $small-font-weight;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-light-grey;
+  }
+
+  &__wrapper {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews {
+    @extend %small-text;
+  }
+
+  &__price-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-size: $small-font-size;
+    line-height: 18px;
+    color: $color-light-grey;
+  }
+
+  &__price-number {
+    font-weight: $big-font-weight;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-light-black;
+  }
+
+  &__link {
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    height: 40px;
```

Comment:
you had better not hardcode height for elements with text/content inside. Use paddings to apply height